packages/core/src/js/sdk.tsx

```diff
@@ -153,12 +153,11 @@ export function init(passedOptions: ReactNativeOptions): void {
  */
 export function wrap<P extends Record<string, unknown>>(
   RootComponent: React.ComponentType<P>,
-  options?: ReactNativeWrapperOptions
+  options?:  Pick<Partial<ReactNativeWrapperOptions>, Exclude<keyof ReactNativeWrapperOptions, 'updateProps' | 'children'>>
```

Comment:
This won't work; we need to update the `profilerProps` in the `ReactNativeWrapperOptions` type.

The `ReactNativeWrapperOptions` don't have `updateProps`.

Reply:
I have updated on https://github.com/getsentry/sentry-react-native/pull/4740/commits/a38cbcb2b699583c1bcc8daf3a3efcac1aef825c to to update the ReactNativeWrapperOptions